### PR TITLE
Implement init stack

### PR DESCRIPTION
**Pull request by Gobleizer**
Sets up Express, React, Babel, and Webpack.

Express serves basic React Web Page.

Includes Hot Loading feature, you can now see your changes to react components in real time, without refreshing the page. Thank you @sbarkerdudley .

PM please review @ahjohnston

### CODE REVIEW

**Comment by Gobleizer** [comment]
[3 Implement Project Skeleton](https://trello.com/c/iazOfFSp/17-3-implement-project-skeleton)

**Comment by ahjohnston** [comment]
[1 review PR](https://trello.com/c/aAEWSuwW/35-1-review-pr)

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const path = require('path');
```
Comment (ahjohnston) [inline webpack.config.js:1]: I don't know anything about webpack configuration--can I trust you that this is all perfect & wonderful?

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
-# jobsite
\ No newline at end of file
+# jobsite
```
Comment (ahjohnston) [inline README.md:1]: so GOOD! very clear, and I love the hotloading option

**Review by ahjohnston — APPROVED** [review]
Beautiful beautiful beautiful

File: webpack.config.js (hunk 1 above)
Comment (maki-q) [reply webpack.config.js:1]: Webpack looks good.

**Review by maki-q — APPROVED** [review]

**Comment by maki-q** [comment]
@ahjohnston I approve merging.

File: README.md (hunk 2 above)
Comment (Gobleizer) [reply README.md:1]: Thank you @sbarkerdudley was the lead on hotloading.

**Pull request merged**